### PR TITLE
[dev-v2.7] Bump system-upgrade-controller chart to 0.10.0

### PR DESCRIPTION
**Pull request by Oats87**
https://github.com/rancher/rancher/issues/40030

See corresponding system-charts change for the changelog of system-upgrade-controller.

This bumps system-upgrade-controller to 0.10.0 which also brings K8s v1.26.0 support with it.

### CODE REVIEW

File: charts/system-upgrade-controller/101.1.0+up0.4.0/values.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+global:
+  cattle:
+    systemDefaultRegistry: ""
+
+systemUpgradeController:
+  image:
+    repository: rancher/system-upgrade-controller
+    tag: v0.9.1
```
Comment (kinarashah) [inline charts/system-upgrade-controller/101.1.0+up0.4.0/values.yaml]: @Oats87 something seems to have gone wrong with the process, I expected this to be v0.10.0?

**Review by kinarashah — COMMENTED** [review]
Build is failing with 
```
time="2023-01-17T21:44:30Z" level=info msg="system-upgrade-controller/101.1.0+up0.4.0 is untracked"
time="2023-01-17T21:44:31Z" level=error msg="The following new assets have been introduced: map[system-upgrade-controller:[101.1.0+up0.4.0]]"
time="2023-01-17T21:44:31Z" level=error msg="The following released assets have been removed: map[]"
time="2023-01-17T21:44:31Z" level=error msg="The following released assets have been modified: map[]"
time="2023-01-17T21:44:31Z" level=error msg="If this was intentional, to allow validation to pass, these charts must be added to the release.yaml."
``` 

could you run `make validate` locally to see if there's any issues?

File: charts/system-upgrade-controller/101.1.0+up0.4.0/values.yaml (hunk 1 above)
Comment (Oats87) [reply charts/system-upgrade-controller/101.1.0+up0.4.0/values.yaml]: Let's give my new commit a try through CI...

**Review by kinarashah — APPROVED** [review]

**Review by aiyengar2 — APPROVED** [review]
$ diff -uNr charts/system-upgrade-controller/101.0.0+up0.3.3 charts/system-upgra
de-controller/101.1.0+up0.4.0

```diff
diff -uNr charts/system-upgrade-controller/101.0.0+up0.3.3/Chart.yaml charts/system-upgrade-controller/101.1.0+up0.4.0/Chart.yaml
--- charts/system-upgrade-controller/101.0.0+up0.3.3/Chart.yaml	2022-12-22 16:12:31
+++ charts/system-upgrade-controller/101.1.0+up0.4.0/Chart.yaml	2023-01-20 09:32:59
@@ -1,18 +1,18 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
-  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.25.0-0'
+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.26.0-0'
   catalog.cattle.io/namespace: cattle-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux,windows
   catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
   catalog.cattle.io/release-name: system-upgrade-controller
 apiVersion: v1
-appVersion: v0.9.1
+appVersion: v0.10.0
 description: General purpose controller to make system level updates to nodes
-home: https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader
+home: https://github.com/rancher/system-charts/blob/dev-v2.7/charts/rancher-k3s-upgrader
 kubeVersion: '>= 1.16.0-0'
 name: system-upgrade-controller
 sources:
-- https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader
-version: 101.0.0+up0.3.3
+- https://github.com/rancher/system-charts/blob/dev-v2.7/charts/rancher-k3s-upgrader
+version: 101.1.0+up0.4.0
diff -uNr charts/system-upgrade-controller/101.0.0+up0.3.3/templates/psp.yaml charts/system-upgrade-controller/101.1.0+up0.4.0/templates/psp.yaml
--- charts/system-upgrade-controller/101.0.0+up0.3.3/templates/psp.yaml	2022-12-22 16:12:31
+++ charts/system-upgrade-controller/101.1.0+up0.4.0/templates/psp.yaml	2023-01-20 09:32:59
@@ -1,3 +1,4 @@
+{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
 apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
@@ -47,3 +48,4 @@
   - kind: Group
     apiGroup: rbac.authorization.k8s.io
     name: system:serviceaccounts:cattle-system
+{{- end }}
diff -uNr charts/system-upgrade-controller/101.0.0+up0.3.3/values.yaml charts/system-upgrade-controller/101.1.0+up0.4.0/values.yaml
--- charts/system-upgrade-controller/101.0.0+up0.3.3/values.yaml	2022-12-22 16:12:31
+++ charts/system-upgrade-controller/101.1.0+up0.4.0/values.yaml	2023-01-20 09:32:59
@@ -5,9 +5,9 @@
 systemUpgradeController:
   image:
     repository: rancher/system-upgrade-controller
-    tag: v0.9.1
+    tag: v0.10.0

 kubectl:
   image:
     repository: rancher/kubectl
-    tag: v1.20.15
+    tag: v1.23.3
```

LGTM

**Pull request merged**